### PR TITLE
Kuiper imager update 

### PR DESCRIPTION
**Pull request by scuciurean**

### CODE REVIEW

**Comment by scuciurean** [comment]
Trigger appveyor build

File: proj_list.json [hunk 1]
```diff
@@ -1,122 +1,149 @@
...
+                                },
+                                {
+                                        "name": "ZCU 102 (rev 1.0)",
+                                        "path": "zynqmp-zcu102-rev10-",
+                                        "daughterboard" : [
+                                                "ad9081-m8-l4",
+                                                "ad9172-fmc-ebz-mode4",
+                                                "ad9361-fmcomms2-3-4",
+                                                "ad9361-fmcomms5",
+                                                "adrv9002",
```
Comment (tfcollins) [inline proj_list.json:116]: ADRV9002 has multiple BOOT.BINs and DTs. Is this handled?

**Comment by scuciurean** [comment]
Chagelog: 
- Add separate windows for flashing and configuring
- Get os list from swupload
- Add azure pipeline
- General UI changes

**Review by rgetz — APPROVED** [review]
looks ok - but some may need to be refactored to send upstream.

**Comment by tfcollins** [comment]
image

Logo is still cut off

Also, what is the difference between the two options?

**Comment by rgetz** [comment]
yeah, I think those look more like "buttons", like the other pages/original package (so end users know what to click).

image

**Comment by scuciurean** [comment]
> image
> 
> Logo is still cut off

Maybe is something related to the fonts
image

image


> Also, what is the difference between the two options?

On the right you can write the .img file to the sd-card. It's like a window for the dd command only. In this way you can write the kuiper image or even the ones here https://github.com/analogdevicesinc/aditof_sdk. 
On the left side you can configure the image: copy the devicetree and binaries to the boot partition and edit startup commands.
Those things are separate so that the imager won't start writing the config.txt file (for kuiper) for the ToF images (where it doesn't exist).

**Comment by rgetz** [comment]
>Maybe is something related to the fonts or the theme you're using

I'm using the Windows version - I don't have fonts/themes.

-Robin

**Pull request closed without merging**